### PR TITLE
Fix dependency change of libp11sgx.so

### PR DESCRIPTION
**Pull request by qiming-007**
Fix dependency change of `libp11sgx.so`, removed libs may change after SGX SDK updating.

### CODE REVIEW

File: cmd/waitcmd.go [hunk 1]
```diff
@@ -22,9 +22,7 @@ const (
 )
 
 var (
-	sgxLibNames = []string{"libp11SgxEnclave.signed.so", "libp11sgx.so",
-		"libsgx_dcap_ql.so.1", "libsgx_enclave_common.so.1",
-		"libsgx_pce_logic.so.1", "libsgx_qe3_logic.so", "libsgx_urts.so"}
+	sgxLibNames = []string{"libp11SgxEnclave.signed.so", "libp11sgx.so"}
```
Comment (zhlsunshine) [inline cmd/waitcmd.go:25]: why need to remove other libs? It should be used below, I suggest that all libs should be checked before the pods get ready.

Comment (qiming-007) [reply cmd/waitcmd.go:25]: Because other lib's name may change after SGX SDK updating. Now `libsgx_urts.so` has been change to `libsgx_urts.so.2`.

Comment (qiming-007) [reply cmd/waitcmd.go:25]: Only the name of `libp11SgxEnclave.signed.so`, `libp11sgx.so` can keep unchanged.

**Review by Chen-Xintong — APPROVED** [review]
LGTM

**Pull request merged**